repository-data/webfiles/src/main/resources/site/apispecification/rapidoc.js

```diff
@@ -0,0 +1,153 @@
+// stick nav js adapted from NHSD frontend
```

Comment:
looks like you can wrap all this in a [self-executing anonymous function](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) to stop everything being available at a global level.